scripts/deprecation_report.py

```diff
@@ -0,0 +1,11 @@
+import sys
```

Comment:
I think this can just be removed. It is sufficient that it runs as part of the unit test.